src/PatientChecking/PatientChecking/Controllers/DashboardController.cs

```diff
@@ -9,10 +11,37 @@ namespace PatientChecking.Controllers
 {
     public class DashboardController : BaseController
     {
+        private readonly IPatientService _patientService;
+        private readonly IAppointmentService _appointmentService;
+
+        public DashboardController(IPatientService patientService, IAppointmentService appointmentService)
+        {
+            _patientService = patientService;
+            _appointmentService = appointmentService;
+        }
+
         public IActionResult Home()
         {
             return View();
         }
+        [HttpGet("[controller]/getDashBoard")]
+        public async Task<IActionResult> GetDashBoardData()
+        {
+            int numOfPatients = await _patientService.GetNumberOfPatients();
+            int numOfPatientsInMonth = await _patientService.GetNumberOfPatientsInCurrentMonth();
+            int numOfAppointment = await _appointmentService.GetNumberOfAppointments();
+            int numOfAppointmentInMonth = await _appointmentService.GetNumberOfAppointmentsInCurrentMonth();
+            int numOfAppointmentInToday = await _appointmentService.GetNumberOfAppointmentsInToday();
```

Comment:
Please create a separate Model to combine all three values needed for the appointment. Those values must be proceeded in one sql command by writing only one service method named appointmentService.GetAppointmentSummary(), to reduce times transfer data between the application and sql server.
Apply the same for patient service.

Reply:
I created separate models for patients and appointments to combine all values needed for each